openwisp_utils/admin.py

```diff
@@ -1,4 +1,8 @@
+from typing import Tuple, List
```

Comment:
is it needed? My local `flake8` check is failing because of this line:

```
$ flake8
./openwisp_utils/admin.py:1:1: F401 'typing.Tuple' imported but unused
./openwisp_utils/admin.py:1:1: F401 'typing.List' imported but unused
```